### DISCUSSION TITLE
Comprovação de trabalho remoto

### DISCUSSION THREAD

**Discussion by KumoMg** [frontendbr/forum#762] [Discussão]
Bom dia galera!

Eu estava lendo um post antigo aqui no fórum sobre ["trabalhar em troca de equity](https://github.com/frontendbr/forum/issues/590)" e me surgiu uma curiosidade, como a galera que trabalhou remotamente prova para um potencial empregador no Brasil essa experiência(visto que não vão ter a experiencia assinada na carteira de trabalho)?

**Comment by sbvorpagel** [comment]
@KumoMg Acho que depende muito da empresa. Noto que é cada vez menor a solicitações de coisas formais como experiência em carteira, faculdade etc. Hoje em dia é muito mais importante um portfólio, contribuições em projetos, github etc.

Claro como falei acima, é de empresa para empresa, mas é notório que essa evolução está acontecendo, ou até já aconteceu.

**Comment by victorferraz** [comment]
@KumoMg ultimamente não existe muito de checar a  experiência em carteira de trabalho, acho que nunca me pediram para olhar minha carteira de trabalho antes de me contratarem. O Linkedin por ser uma rede social, pública é mais difícil as pessoas escreverem algo que não é verdade.
Além dos exemplos citados no caso do @sbvorpagel existe também o famoso testes que é passado antes de ser contrato ou antes de ser fechado como sócio. Com algumas linhas de código é possível ver qual o grau de  conhecimento do futuro sócio.

**Comment by felipefialho** [comment]
Na conversa, com código e fazendo os testes de conhecimentos exigidos pelo recrutador 😆
